src/extension.ts

```diff
@@ -1460,7 +1465,82 @@ async function execDraftCreateApp(appName : string, pack? : string) : Promise<vo
     }
 }
 
-function draftCreateResult(sr : ShellResult, hadPack : boolean) {
+async function execDraftCreateDebug() {
+
+    const pack = "github.com/radu-matei/draft-packs/packs/node-debug/"
+
+    if (vscode.workspace.rootPath === undefined) {
+        vscode.window.showErrorMessage('This command requires an open folder.');
+        return;
+    }
+    if (draft.isFolderMapped(vscode.workspace.rootPath)) {
+        vscode.window.showInformationMessage('This folder is already configured for draft. Run draft up to deploy.');
+        return;
+    }
+    if (!(await draft.checkPresent())) {
+        return;
+    }
+    const proposedAppName = path.basename(vscode.workspace.rootPath);
+    const appName = await vscode.window.showInputBox({ value: proposedAppName, prompt: "Choose a name for the Helm release" });
+    if (appName) {
+        await execDraftCreateApp(appName, pack);
+    }
+}
+
+async function execDraftUpDebug() {
+    const draftPath = await draft.path();
+    const term = vscode.window.createTerminal('draft logs', `bash`, ['-c', `${draftPath} logs ; bash`]);
+    term.show(true);
+
+    vscode.window.showInformationMessage("Starting draft up")
+    await draft.invokeAsync('up').then((result) => {
+        vscode.window.showInformationMessage("Finished draft up")
+        execDraftDebug() 
+    })
+
+}
+
+async function execDraftDebug() {
+
+    // exits with code 0 when it found a pod to connect to
+    // replicaCount is set to 1 in the debug pack, because with rolling updates, 
+    // it will find a running pod soon to be terminated, connect to it and then it gets killed
+    // TODO - fix this once helm install --wait is functional
+    await draft.invokeAsync('connect --wait')
+
+    const draftPath = await draft.path();
+
+    // TODO - make ports parameters
+    const term = vscode.window.createTerminal('draft connect', `bash`, ['-c', `${draftPath} connect -p 8080:8080 -p 9229:9229 ; bash`]);
+    term.show(true);
+    
+    vscode.window.showInformationMessage("Attaching debugger...")
+
+    // we know there is a ready pod, this shoud be enough time for 
+    // the portforward stuff to create the connection 
+    // TODO - how can this be avoided? what happens if network is busy?
+    await sleep(5000)
+
+    const debugConfiguration = {
+        type: 'node',
+        request: 'attach',
+        name: 'Attach to Process',
+        address: 'localhost',
+        // TODO - make this configurable - part of draft.toml?
+        port: 9229,
+        localRoot: vscode.workspace.rootPath,
+        remoteRoot: '/usr/src/app'
```

Comment:
Pardon my ignorance - is this directory part of the Draft spec, so guaranteed to work?